web_src/css/editor.css

```diff
@@ -12,6 +12,12 @@
   border-bottom: 0;
 }
 
+.repo-editor-header {
```

Comment:
Why we need this? If I understand correctly, the code is always:

```
<div repo-editor-header>
   <div fitted item treepat>
   </div>
</div>
```

?

Reply:
This is related to the refactor from previous `ui secondary menu` which was adding these rules. If it were left in place, the `.breadcrumb a:hover` rule would have to have an `!important`, but I circumvented it with the replacment.

Reply:
Actually, I was able to remove `.breadcrumb a:hover` as well. Still, the menu removal is necessary as otherwise a important rule would be needed because `.menu` removes `text-decoration` on `a`, and I generally think it is unsuitable for this element.

Reply:
I mean , could we just only use one div ? eg: `<div fitted item treepat>`, no `<div repo-editor-header>` ?

Reply:
Can probably remove one wrapper, yes.

Reply:
Wrapper removed, visual output is the same.